samples/drivers/lora_send/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  description: Demonstration of LoRa Send functionality
+  name: LoRa Send Sample
+tests:
+  test:
+    platform_whitelist: 96b_wistrio
+    tags: LORA
```

Comment:
move the tags to under the sample: block, you can also add "lora" tag together with "LORA" just in case the tag is case-sensitive.

Reply:
> move the tags to under the sample: block

You mean as below?

```
sample:
  description: Demonstration of LoRa Send functionality
  name: LoRa Send Sample
tests:
  sample.driver.lora_send:
    platform_whitelist: 96b_wistrio
    tags: LORA
```
> you can also add "lora" tag together with "LORA" just in case the tag is case-sensitive.

Hmm. I think it makes sense to use just `lora`

Reply:
I meant something like this
```
sample:
   description: Demonstration of LoRa Send functionality
   name: LoRa Send Sample
   tags: lora
tests:
   sample.driver.lora_send:
     platform_whitelist: 96b_wistrio
```
So that if we add new tests, then the tags are automatically set for them.

Reply:
Cool, thanks for the info. The reason I asked for the clarification is, I couldn't find `tags` under `sample` section in any existing samples.

Reply:
@jukkar Looks like CI is failing because of this change.

Reply:
Ah, indeed. The tags: needs to be under common: like this
```
common:
  tags: lora
sample:
   description: Demonstration of LoRa Send functionality
   name: LoRa Send Sample
tests:
   sample.driver.lora_send:
     platform_whitelist: 96b_wistrio

```
Sorry for confusing advice, just came back from vacation :)